app/grants/admin.py

```diff
@@ -410,3 +410,4 @@ class GrantCollectionAdmin(admin.ModelAdmin):
 admin.site.register(GrantCLR, GrantCLRAdmin)
 admin.site.register(GrantCollection, GrantCollectionAdmin)
 admin.site.register(GrantStat, GeneralAdmin)
+admin.site.register(GrantBrandingRoutingPolicy)
```

Comment:
To register the model in the admin, you'll need to create a `GrantBrandingRoutingPolicyAdmin` class in this file and register it like the other model classes